search_service/config.py

```diff
@@ -19,7 +21,12 @@ class LocalConfig(Config):
     TESTING = False
     STATS = False
     LOCAL_HOST = '0.0.0.0'
-    ELASTICSEARCH_ENDPOINT = 'http://{LOCAL_HOST}:9200'.format(LOCAL_HOST=LOCAL_HOST)
+    ELASTICSEARCH_PORT = '9200'
+    ELASTICSEARCH_ENDPOINT = os.environ.get('ELASTICSEARCH_BASE',
```

Comment:
Nit: How about use the same variable name? `ELASTICSEARCH_ENDPOINT` instead of `ELASTICSEARCH_BASE`?

Reply:
PR updated :)